modules/conversion-center/assets/js/editor/hooks/ui/editor/document/select.js

```diff
@@ -0,0 +1,34 @@
+import After from 'elementor-api/modules/hooks/ui/after';
+
+export class AfterSelect extends After {
+	getCommand() {
+		return 'document/elements/select';
+	}
+
+	getId() {
+		return 'prevent-container-selection';
+	}
+
+	getConditions( args ) {
+		let type = args?.container?.document?.config?.type;
+		if ( ! type ) {
+			type = args?.containers[ 0 ]?.document?.config?.type;
+		}
+
+		return 'contact-buttons' === type;
+	}
+
+	apply( args ) {
+		const { container: { type = '' } } = args;
+		switch ( type ) {
+			case 'section':
+			case 'container':
+				$e.run( 'document/elements/deselect', args );
```

Comment:
@nicoladj77 Can you rather just avoid the origin selection instead of selecting and de-selecting?

Reply:
@louiswol94 that looks a lot more complex because I think it's part of the import process. I also think it's cleaner because it actually prevents any possibility of selecting a section or a container.

Reply:
Okay I get you, if the import process selects this automatically, then it's ok to de-select it here with a Hook. nice.